src/main/resources/staff/templates/email/ccjRequestSubmitted/subject.txt

```diff
@@ -1 +1,5 @@
+{% if admissionResponse == true %}
```

Comment:
Can we please use conditional operator only on the changing context. probably use:
```
Civil money claims: {{ claimReferenceNumber }} {{ claimantName }} v {{ defendantName }} {{ admissionResponse ? "judgment by admission requested" : "default judgment requested" }}
```

Reply:
Yes that looks more tidier